### ISSUE TITLE
FK query fail v0.11.2

### ISSUE THREAD

**Issue by Fauny** [balderdashy/sails-mongo#293]
Model parent.js
```javascript
module.exports = {
    attributes: {
        name: {
            type: 'string',
            required: true
        },
        children: {
            collection: 'child',
            via: 'parent'
        }
    }
};
```
Model child.js
```javascript
module.exports = {
    attributes: {
        name: {
            type: 'string',
            required: true
        },
        parent: {
            model: 'parent'
        }
    }
};
```
and this is my query, it works fine on sails-disk, but failed here
```javascript
sails.models.parent.findOne({name: 'aaa'}).exec(function (e,p){
	if (p){
		console.log('parent found !');

		// this returns undefined, but works on localDiskDb
		sails.models.child.find({parent: p.id}).exec(console.log); 

		// this returns undefined too
		sails.models.child.find({parent: p._id}).exec(console.log);
	}
})
```

any suggestion? pls.

**Comment by sailsbot** [comment]
Thanks for posting, @Fauny.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
